t/lib/OpenQA/Test/FullstackUtils.pm

```diff
@@ -159,22 +159,22 @@ sub wait_for_developer_console_contains_log_message {
             && $log =~ $regex_closed)
         {
             fail('web socket connection closed prematurely, was waiting for ' . $diag_info);
+            note(
+"developer console contained at this point:\n$log\n(considering only chars after index $position_of_last_match)"
```

Comment:
I wonder how tidy let this one pass :) I personally find the message (and the one in L174) a bit too long code-style wise. 
![But that's just my opinion](https://user-images.githubusercontent.com/229240/49406671-8545ea80-f756-11e8-98cd-52fc5c55f9bc.png)
)

Reply:
That is how tidy made it...
---
About the long log messages: I've checked and neither the log of the developer fullstack test or the regular fullstack test is truncated. The regular fullstack test is almost not affected at all. In my opinion it is quite useful to see what's going on as long as the developer console is open (especially on Travis where you have otherwise no means of following the test execution).

Reply:
removed the long lines

Reply:
@Martchus yeah, guessed something like this with travis :)